demos/common/python/openvino/model_zoo/model_api/models/instance_segmentation.py

```diff
@@ -0,0 +1,293 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import numpy as np
+
+from .image_model import ImageModel
+from .utils import RESIZE_TYPES, load_labels, nms, pad_image
+
+
+class MaskRCNNModel(ImageModel):
+    def __init__(self, model_adapter, prob_threshold=0.5, labels=None, keep_aspect_ratio=False):
+        super().__init__(model_adapter)
+        self.resize_type = 'fit_to_window' if keep_aspect_ratio else 'standard'
+        self.resize = RESIZE_TYPES[self.resize_type]
+        self.prob_threshold = prob_threshold
+        self._check_io_number((1, 2), (3, 4, 5, 8))
+        self.type = 'segmentoly' if len(self.inputs) == 2 else 'mask_rcnn'
```

Comment:
```suggestion
        self.is_segmentoly = len(self.inputs) == 2
```
The `MaskRCNNModel` wrapper mostly consists of `self.type == 'segmentoly'` conditions, so it would be preferably to create a variable containing this information and use it

Reply:
fixed in the whole script.